roles/pulp-devel/templates/alias.bashrc.j2

```diff
@@ -146,7 +146,11 @@ pyclean () {
 _pyclean_help="Cleanup extra python files"
 
 pfixtures () {
-    podman run --rm -p 8000:80 quay.io/pulp/pulp-fixtures:latest
+    cat ~/.config/pulp_smash/settings.json | \
+    jq "setpath([\"custom\",\"fixtures_origin\"]; \"http://localhost:8000/fixtures/\")" > temp.json
+    cat temp.json > ~/.config/pulp_smash/settings.json
+    rm temp.json
```

Comment:
Let's not use `temp.json` in the current directory, which could be unwriteable. Let's use a path like `/tmp/temp_pfixtures.json`.

Reply:
Additional justification: I also often have files named like `temp.json` in my directories as I am testing stuff.